schema/shared/src/main/scala/zio/blocks/schema/Reflect.scala

```diff
@@ -99,7 +99,9 @@ sealed trait Reflect[F[_, _], A] extends Reflectable[A] { self =>
     case _                   => false
   }
 
-  def fromDynamicValue(value: DynamicValue)(implicit F: HasBinding[F]): Either[SchemaError, A]
+  def fromDynamicValue(value: DynamicValue, trace: List[DynamicOptic.Node])(implicit
```

Comment:
Such a detail like `trace` should be hidden in a private function and not accessible to the top-level.